src/index.ts

```diff
@@ -400,6 +406,7 @@ const fileHandler: ReqListener = async (req, res) => { //? File handler
 			const stats = fs.statSync(faviconPath)
 			const etagValue = etag(stats)
 			res
+			.setHeader('X-Powered-By', 'urobbyu/serve')
```

Comment:
Not the right place to put it. This header looks like it should be sent regardless of the contents of the response.